src/ResourceManager/Resources/Commands.ResourceManager/Cmdlets/Implementation/Policy/PolicyAssignmentCmdletBase.cs

```diff
@@ -0,0 +1,57 @@
+﻿// ----------------------------------------------------------------------------------
```

Comment:
No file in this module should be updated

Reply:
Same as above.